.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,104 @@
+name: Mac and Windows Builds
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+        python-version: ['3.5', '3.6', '3.7']  # We build 3.5 on Azure Pipelines
+        architecture: ['x86', 'x64']
+        exclude:
+          # Don't build 32-bit on Mac
+          - os: macos-latest
+            architecture: x86
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # This LOOKS like it is checking out 'master', but it is using the 'master' version of the checkout action
+      # It is actually checking out the most recent version on this branch
+      - name: Checkout
+        uses: actions/checkout@master
+
+      # Visual Studio 2019 (windows-latest) doesn't include these files for some reason??
+      # Even stranger, it really only affects Python 3.5, but we run it across all versions for consistency
+      # https://social.msdn.microsoft.com/Forums/vstudio/en-US/06675e6d-9a0b-46f5-8de0-10237fab1ece/link-error-lnk1158?forum=vcgeneral
+      - name: Fix Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          Copy-Item "C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\rc.exe" -Destination "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN"
+          Copy-Item "C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\rcdll.dll" -Destination "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN"
+        shell: powershell
+
+      - name: Setting up Python
+        uses: actions/setup-python@v1.1.1
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: ${{ matrix.architecture }}
+
+      - name: Updating pip
+        run: python -m pip install --upgrade pip
+        shell: bash
+
+      - name: Installing requirements
+        run: |
+          pip install -r requirements.txt
+          pip install -r build_tools/build_requirements.txt
+        shell: bash
+
+      - name: Building wheel
+        run: make bdist_wheel
+        shell: bash
+
+      - name: Installing generated wheel
+        run: pip install --pre --no-index --find-links dist/ pmdarima
+        shell: bash
+
+      - name: Running unit tests
+        run: |
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            export PMD_MPL_BACKEND=TkAgg
+          fi
+          pytest --showlocals --durations=20 --mpl --mpl-baseline-path=etc/pytest_images --pyargs pmdarima --benchmark-skip
```

Comment:
You can change this to:

```
pytest --showlocals --durations=20 --pyargs pmdarima --benchmark-skip
```

We no longer need MPL image comparisons in our testing.

Reply:
In fact, if that's still in the Azure pipelines, we should remove that too.

Reply:
Fixed on ADO (conda) and Github Actions with 006a582415861cbfe9bac8104990770b316bfdce